package.json

```
@@ -3,5 +3,13 @@
     "@actions/core": "^1.2.7",
     "@actions/exec": "^1.1.0",
     "@actions/tool-cache": "^1.6.1"
+  },
+  "devDependencies": {
+    "@vercel/ncc": "^0.31.1",
+    "eslint": "^8.2.0"
+  },
+  "scripts": {
+    "eslint": "eslint",
+    "ncc": "ncc"
```

Added these for my own convenience. Maybe they should be replaced with something like this?
```suggestion
    "fix": "eslint --fix index.js",
    "build": "ncc build"
```

---

`eslint` would make for a nice candidate for a workflow. Same for `ncc`, but that's trickier to do properly.

Also, these additions probably warrant some documentation in `README.md`.

---

I'll add a README 😂

---

I would go with `lint` and `build` as script names.

---

I didn't notice that it also fixed it. In that case `fix` seems fine, although it's not specific what it fixes exactly. 

Anyway, just a thought since I don't actually use this repo 😅